tests/test_schema_validation.py

```diff
@@ -188,6 +188,15 @@ def test_invalid_question_titles_object(self):
         self.assertEqual(errors[1]['message'], 'Schema Integrity Error. The answer id - behalf-of-answer-fake in the '
                                                '"when" clause for what-gender-question does not exist')
 
+    def test_invalid_survey_id_whitespace(self):
+
+        file = 'schemas/test_invalid_survey_id_whitespace.json'
+        json_to_validate = self.open_and_load_schema_file(file)
+
+        errors = self.validator.validate_schema(json_to_validate)
+
+        self.assertEquals(errors.get('message'), "'lms ' does not match '^[0-9a-z]+$'")
```

Comment:
Where does the message come from in this instance? I think it could be changed to something more readable like "Please remove all whitespaces from survey_id 'lms '"